packages/webdriverio/test/axe-webdriverio.spec.ts

```diff
@@ -1411,13 +1411,8 @@ describe('@axe-core/webdriverio', () => {
 
           assert.notEqual(title, 'Error');
 
-          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-          // @ts-ignore
-          delete client.createWindow;
-          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-          // @ts-ignore
-          client.createWindow = () => {
-            return { handle: null };
+          client.createWindow = (type: 'tab' | 'window') => {
+            return Promise.resolve({ handle: '', type });
```

Comment:
Just replacing the method with a properly typed stub to match the underlying signature.